dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/informed-consent-template-verbal-covid.md

```diff
@@ -0,0 +1,73 @@
+if so, we will ask you again whether you consent to participate and you may choose whether you will consent to participate at that time.
+
+### Benefits
+**[LIST ALL BENEFITS FROM THE STUDY TO PARTICIPANTS]**
+
+### Risks/Discomforts
+As with all research, there is a chance that confidentiality could be compromised;
+however, we are taking precautions to minimize this risk. **[LIST ALL RISKS FROM THE STUDY TO PARTICIPANTS]**
+
+_If in-person survey add following_
+As you are probably aware, the world has experienced an outbreak of a novel disease in 2020, called the COVID-19. It is highly contagious, transmitted mostly through close contact with an infected person in an indoor space. Symptoms vary from one person to the other, but often include headache, fever, dry cough. It can be particularly dangerous for vulnerable populations such as older people (60+), pregnant women or anyone with serious chronic conditions.
+While I and my team members have no symptoms and are not aware of any close contact with a COVID+ person, there is a risk of contamination or spread of COVID-19. We are minimizing the risk of this spread by following the most effective known mitigation strategies, as recommended by the **[Country]** health ministry and the World Health Organization:
+•	maintaining a distance of 2m/6ft
+•	wearing masks and gloves throughout
+•	providing you with a mask
+•	remaining outside  
+•	<any other protocols>
+We are also willing to follow any extra measure that would make you feel more comfortable when taking the survey.
+If in the next couple of weeks, we realize that we might have put you at risk (discovering for instance that a member of our team contracted the COVID-19 or has been in contact with a sick person), we will immediately contact you and refer you to the closest health facility to get tested. In the meantime, here / we will send you literature on COVID-19, its symptoms, how to limit transmission and what to do if you think you might have COVID-19.
+
+
+
+### Confidentiality
+Your study data will be handled as confidentially as possible.
+If results of this study are published or presented, individual names, and other personally identifiable information will not be used.
+No identifying information will be shared with any third party.
+We will ask for your consent in this study, and document it on this tablet.  
+
+To minimize the risks to confidentiality, we will keep all identifying information stored in encrypted files.
+Your responses will be numbered, and the code linking your name with your responses will also be stored in encrypted files on
+password-protected computers.
+Only trained interviewers and researchers from **[LIST OF ORGANIZATIONS]** will have access to these files.  
+
+When the research is completed, we may save the information for use in future research done by this research team.
+We will retain these records for up to **[NUMBER OF YEARS]** years after the study is over.
```

Comment:
I wonder if we should remove this since we do not have a specific data destruction protocol?

Reply:
actually i wonder if we might remove this whole paragraph since we already mention possibility of follow-up earlier ...

motivation is just that now the consent is pretty long so trying to think if we might streamline other bits that are not very relevant